Header header

geometry_msgs/Vector3 LLA
geometry_msgs/Vector3 UTM
geometry_msgs/Vector3 velocity